.gitignore

```diff
@@ -17,4 +17,6 @@
 /tmp
 
 # Ignore Heroku app binstubs
-/bin/*
\ No newline at end of file
+/bin/*
+
+/Gemfile.lock
```

Comment:
it's standard for gems to ignore the lockfile, since apps that depend on this gem will most commonly resolve from scratch (the `bundle gem` template adds the lock to gitignore)